LIBIIF-32. Download button added for full resolution images.

siddharthbhagwan opened the pull request:
- Button moved to HUD panel on the left with other image manipulation tools

https://issues.umd.edu/browse/LIBIIIF-32

siddharthbhagwan committed:
LIBIIF-32. Download button added for full resolution images.

- Button moved to HUD panel on the left with other image manipulation tools

https://issues.umd.edu/browse/LIBIIIF-32

peichman-umd commented on js/src/widgets/hud.js:
```diff
@@ -293,6 +334,11 @@
                                   '</a>',
                                   '</div>',
+                                  '<div class="mirador-download-control">',
+                                  '<a class="mirador-download-button hud-control" role="button" title="{{t "downloadButton"}}" aria-label="{{t "downloadButton"}}" download="image">',
```
Since Firefox will only honor the `download` attribute when the image and the script have the same origin, add a `target="_blank"` attribute to the link so that Firefox will at least open a new tab instead of just following the link and losing the user's place in the Mirador viewer.

siddharthbhagwan committed:
LIBIIIF-32. rget attribute added for Firefox support.

 - See https://github.com/umd-lib/mirador/pull/11/files#r153879451

https://issues.umd.edu/browse/LIBIIIF-32
